src/OpenConext/Profile/Tests/fixture/arp-response.json

```diff
@@ -66,5 +66,12 @@
     {
       "value": "*"
     }
+  ],
+  "urn:mace:dir:attribute-def:eduPersonAffiliation": [
```

Comment:
This attribute is already in the fixture, may I suggest you pick a different attr name here?

Also, since we allow the `motivation` field too, that (sh|c)ould be covered in this test too?

Reply:
@MKodde As I have no clue what attributes really are, can you give me a suggestion? Or can it be anything?

Reply:
And sure, I can add motivation as well (as this wasn't subject of this use case, I did not think about this)

Reply:
The attributes are released by the IdP and they contain the data that describe the user. For example there is a mail attribute, a username, commonname, lastname.. You name e'm, there is an attribute for that. And if it does not exist, we can add them to the attribute definition.

I do not know all attributes by memory. I suggest you look in the attribute dictionary file. And pick one from there?

@see: vendor/surfnet/stepup-saml-bundle/src/Resources/config/saml_attributes.yaml

Reply:
I added "motivation" to one of the testcases